src/node/setupServer.ts

```diff
@@ -1,5 +1,5 @@
-import { ClientRequestInterceptor } from '@mswjs/interceptors/lib/interceptors/ClientRequest'
-import { XMLHttpRequestInterceptor } from '@mswjs/interceptors/lib/interceptors/XMLHttpRequest'
+import { ClientRequestInterceptor } from '@mswjs/interceptors/lib/interceptors/ClientRequest/index.js'
+import { XMLHttpRequestInterceptor } from '@mswjs/interceptors/lib/interceptors/XMLHttpRequest/index.js'
```

Comment:
we should really just update interceptors to expose an esm build.  

I started on that over the weekend, and have some thoughts on how we could do that, but I think we'll need to probably remove `debug` as a dependency there - since it relies on `tty` and `os`, and i'm having issues getting those properly bundled to a browser compat version, that multiple build tools like.

Reply:
We need to look into some `debug` recipes perhaps. It has browser support, so it should run in the browser. Perhaps we can do some import map for this dependency to ensure that in the browser it always imports the browser dist.